jina/hubble/hubio.py

```diff
@@ -533,7 +533,7 @@ def _get_prettyprint_usage(self, console, executor_name, usage_kind=None):
     @disk_cache_offline(cache_file=str(_cache_file))
     def fetch_meta(
         name: str,
-        tag: Optional[str] = None,
```

Comment:
According to the code, it seems to be an Optional argument indeed

Reply:
According to the implementation of `disk_cache_offline`,  the optional argument is not included in the cache key. That's the reason I made this revision to cache the meta info regarding specific tag.

Reply:
The `tag` is not optional in the function's signature. But `self.fetch_meta('dummy_exe', None)` will be executed when pulling with `uses='jinahub://dummy_exec'` without specific tag.

Reply:
Please, add note as discussed